jsonParser.go

```diff
@@ -0,0 +1,195 @@
+	IsReferencedByCount int              `json:"is-referenced-by-count"`
+	Issn                []string         `json:"ISSN"`
+	IssnType            []IssnType       `json:"issn-type"`
+	Issue               string           `json:"issue"`
+	Issued              Issued           `json:"issued"`
+	JournalIssue        *JournalIssue    `json:"journal-issue"` // Gap
+	Language            *string          `json:"language"`      //Gap
+	Link                *[]Link          `json:"link"`          // Gap
+	Member              string           `json:"member"`
+	OriginalTitle       *[]any           `json:"original-title"` // 2021
+	Page                string           `json:"page"`
+	Prefix              string           `json:"prefix"`
+	Published           *Published       `json:"published"`        // Gap
+	PublishedOnline     *PublishedOnline `json:"published-online"` // Gap
+	PublishedOther      *PublishedOther  `json:"published-other"`  // Gap
+	PublishedPrint      *PublishedPrint  `json:"published-print"`
+	Publisher           string           `json:"publisher"`
+	Reference           *[]Reference     `json:"reference"` // Gap
+	ReferenceCount      int              `json:"reference-count"`
+	ReferencesCount     int              `json:"references-count"`
+	Relation            *Relation        `json:"relation"` // 2021
+	Resource            Resource         `json:"resource"`
+	Score               float64          `json:"score"`
+	ShortContainerTitle *[]string        `json:"short-container-title"` // 2021
+	ShortTitle          *[]any           `json:"short-title"`           // 2021
+	Source              string           `json:"source"`
+	Subject             []string         `json:"subject"`
+	Subtitle            *[]any           `json:"subtitle"` // 2021
+	Title               []string         `json:"title"`
+	Type                string           `json:"type"`
+	URL                 string           `json:"URL"`
+	UpdatePolicy        *string          `json:"update-policy"` // Gap
+	Volume              string           `json:"volume"`
+	License             []License        `json:"license"`
+	AlternativeID       []string         `json:"alternative-id"`
+}
+
+type Indexed struct {
+	DateParts [][]int   `json:"date-parts"`
+	DateTime  time.Time `json:"date-time"`
+	Timestamp int64     `json:"timestamp"`
+}
+type ContentDomain struct {
+	Domain               []string `json:"domain"`
+	CrossmarkRestriction bool     `json:"crossmark-restriction"`
+}
+type PublishedPrint struct {
+	DateParts [][]int `json:"date-parts"`
+}
+type Created struct {
+	DateParts [][]int   `json:"date-parts"`
+	DateTime  time.Time `json:"date-time"`
+	Timestamp int64     `json:"timestamp"`
+}
+type Affiliation struct {
+	Name string `json:"name"`
+}
+type Author struct {
+	Given       *string        `json:"given"`
+	Family      *string        `json:"family"`
+	Sequence    *string        `json:"sequence"`
+	Affiliation *[]Affiliation `json:"affiliation"`
+}
+type PublishedOnline struct {
```

Comment:
Indexed, PublishedPrint, Created, Deposited, Issued, PublishedOther, Published, Start and PublishedOnline should be the same type